### ISSUE TITLE
Find out how to publish to Sonar Update Center

### ISSUE THREAD

**Issue by arturbosch** [arturbosch/sonar-kotlin#6]
https://docs.sonarqube.org/display/SONAR/Update+Center

**Comment by hameno** [comment]
Seems like https://docs.sonarqube.org/display/DEV/Deploying+to+Update+Center describes the process

**Comment by ablenesi** [comment]
Is there any plan on releasing to Sonar Update Center in the near future?

**Comment by arturbosch** [comment]
No, no plan yet. This project is still proof of concept and should be used on your own risk. In my limited time detekt itself gets more attention (near 1.0 release). After that I plan to work on `sonar-kotlin` again.

**Comment by arturbosch** [comment]
detekt-sonar is now part of the marketplace :partying_face:
